api.go

```diff
@@ -53,6 +53,11 @@ func main() {
 
 	r.Handle("/sessions/{sessionId}/ws/", server)
 
+	// Reverse proxy
+	proxyHandler := handlers.NewMultipleHostReverseProxy()
+	r.Host(`{node}-{port:[0-9]*}.play-with-docker.com`).Handler(proxyHandler)
+	r.Host(`{node}.play-with-docker.com`).Handler(proxyHandler)
+
```

Comment:
> Maybe this should be an optional (configurable) component i.e.
> 
> on / off

What would be the reason to turn it off?. 





> domain name (not everyone will test / use this on "play-with-docker.com"

I agree, and I'd also like to test this locally. Can we remove the domain from the URL and assert by regex matching @xetorthio ?